libs/checkpoint-sqlite/langgraph/checkpoint/sqlite/__init__.py

```diff
@@ -318,7 +318,9 @@ def list(
         ORDER BY checkpoint_id DESC"""
         if limit:
             query += f" LIMIT {limit}"
-        with self.cursor(transaction=False) as cur:
+        with self.cursor(transaction=False) as cur, self.cursor(
```

Comment:
two cursors?

Reply:
not sure what's a better way to do it other than just put everything into a single query and join aggregated writes (not sure if doable in sqlite)